### DISCUSSION TITLE
Checking entailment with Jena?

### DISCUSSION THREAD

**Discussion by samiMHMD** [apache/jena#1473] [Q&A]
Is there a way with Jena to check the entailment of a graph (rdf) against an ontology? (like in TestOWLConsistency for consistency)
 By entailment I mean:
Definition: A collection O of abstract OWL ontologies and axioms and facts entails an abstract OWL ontology or axiom or fact O' with respect to a datatype map D if each interpretation with respect to map D that satisfies each ontology and axiom and fact in O also satisfies O'.

Thanks in advance,
Sami

**Comment by afs** [comment]
Not an answer:

If you don't get an answer here, you can also try the Jena users list [users@jena.apache.org](mailto:users@jena.apache.org). Not everyone is subscribed to every channel.

Joining list information:
https://jena.apache.org/help_and_support/index.html
